zppy/utils.py

```diff
@@ -59,6 +61,18 @@ def getTasks(config, section_name):
             if get_active_status(task):
                 tasks.append(task)
 
+    # Replace `$USER` from the cfg file with the actual username
+    try:
+        machine_info = MachineInfo()
+        username = machine_info.config.get("web_portal", "username")
+    except ValueError:
+        # For unit tests
+        username = os.environ.get("USER")
```

Comment:
@xylar It seems like `os.environ.get("USER")` would always be simpler than using Mache for the username. Is there a reason it'd be better to use Mache for the username?

Reply:
Oh, okay, that's fine.  There is a good reason to have it as a config option in `mache`, which is that it can then be referred to within other config options.  But I completely agree in this context.